ocamltest/tsl_lexer.mll

```diff
@@ -30,7 +30,17 @@ let blank = [' ' '\009' '\012']
 let identchar = ['A'-'Z' 'a'-'z' '_' '.' '-' '\'' '0'-'9']
 let num = ['0'-'9']
 
-rule token = parse
+rule is_test = parse
+  | blank * { is_test lexbuf }
+  | newline { is_test lexbuf }
```

Comment:
It probably makes no different for your use-case but I would still like to see a `Lexing.new_line lexbuf` in the `newline` case. Otherwise there is a bug waiting to happen if we (do something with positions, as in your other PR and) decide to factorize the code later.

Reply:
Done.